composer.json

```diff
@@ -23,7 +23,7 @@
         "doctrine/doctrine-cache-bundle": "~1.1",
         "friendsofsymfony/http-cache": "~1.4",
         "jms/serializer-bundle": "^1.4",
-        "friendsofsymfony/rest-bundle": "^1.6",
+        "friendsofsymfony/rest-bundle": "^2.2",
```

Comment:
Do we use any specific features starting from 2.1? If no I would write `^2.0`, in order to be compatible with more different versions.